### DISCUSSION TITLE
[Kanban] Support for other story views

### DISCUSSION THREAD

**Discussion by AshRubigo** [BurningTreeC/tiddlywiki-muuri#11] [Ideas]
Edit: This specific issue seems to be fixed by @BurningTreeC for version 0.1.13. Another related issue is at https://github.com/BurningTreeC/tiddlywiki-muuri-kanban/issues/2.

Currently when using the 'zoomin' story view and multiple boards, moving a card from one list to another within same board, causes the lists of all other boards to be cleared. I've only tested this with zoomin.

Technically I suppose it's not a bug, but as a user the expected behaviour is that if I have Muuri and Muuri-Kanban installed, that Muuri-Kanban would work using any story view.

This is very precarious, considering it would be easy to forget about this limitation of Muuri-Kanban and switch to a different story view, then do a lot of work before realising that all but 1 kanban board have had their lists cleared.

**Comment by BurningTreeC** [comment]
Hello @AshRubigo , it's true that I didn't test the Muuri Kanbans with the zoomin storyview. I'll have to look deeper into the problems that arise from that. Thanks for giving me an example where it causes problems. I'll have a look how I can fix this
